### PR TITLE
Add functionality to open file after duplicate + configuration option

### PR DESCRIPTION
**Pull request by dougshamoo**
@mrmlnc Hi there, thanks for all your work on this extension. I went ahead and worked up an initial implementation for #39, as I think this is an important feature (and I've started editing the wrong file one too many times :-P). Let me know what you think.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,17 @@
       "editor/title/context": [{
         "command": "duplicate.execute"
       }]
+    },
+    "configuration": {
+      "type": "object",
+      "title": "VSCode-Duplicate configuration",
```
Comment (mrmlnc) [inline package.json]: Please, replace title field to `Duplicate Action Configuration` for consistency with other plugins.

**Review by mrmlnc — APPROVED** [review]

**Comment by mrmlnc** [comment]
Hello, @dougshamoo,

Thanks for PR! I really appreciate it 🎉

Sorry for delay I was on vacation without access to the internet. Please, fix one comment and I'll Merge your changes.

**Comment by dougshamoo** [comment]
No problem, always willing to help out with tools that I use myself. Updated. Thanks again for your work on this. Let me know if you need any more help.

**Comment by mrmlnc** [comment]
Thanks for your help! I'll publish extension tomorrow with fix for #40.

**Pull request merged**